packages/forms-web-app/jest.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   coverageReporters: ['json', 'html', 'text', 'text-summary'],
   coverageThreshold: {
     global: {
-      branches: 95,
+      branches: 94,
```

Comment:
I can't explain this.

When running all the tests (`npm run test:watch`) I see the following lines are not covered:

```
  task.service.js                                  |     100 |    93.55 |     100 |     100 | 147-161 
```

Yet when I run the tests explicitly against that file, I see:

```
task.service.js                                  |   53.85 |    22.58 |      25 |      54 | 8-51,61-82,140
```

This file is not 100% covered in isolation - other tests cover parts of this. 

I have added explicit coverage for the affected lines, which is why I am confused why it doesn't reflect this in both variants. 

Feel free to pull, run locally, and see if you can offer an explanation. I am stumped.